insights/parsers/fc_match.py

```diff
@@ -40,7 +43,7 @@
 @parser(fc_match)
 class FCMatch(Parser):
     """
-    Class to parse command `/usr/bin/fc-match -sv 'sans:regular:roman' family fontformat`.
+    Class to parse command `/bin/fc-match -sv 'sans:regular:roman' family fontformat`.
     This object provides the ' __getitem__' and '__iter__' methods to allow it to
     be used as a list to iterate over the ``data`` data, e.g.::
```

Comment:
Before I merge it, @zhangyi733 - do you still want to remove the redundant `data` here?